docs/docs/09.3-environments.md

```diff
@@ -18,63 +39,35 @@ Example:
 ```java
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.MalformedURLException;
-import java.net.URL;
+import java.io.FileReader;
 
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
 
-import org.apache.commons.io.IOUtils;  // https://commons.apache.org/proper/commons-io/download_io.cgi
-
 public class ReactRender
 {
   public static void main(String[] args) throws ScriptException, IOException {
     ScriptEngine nashorn = new ScriptEngineManager().getEngineByName("nashorn");
 
-    nashorn.eval(getScript("https://fb.me/react-15.0.1.js"));
-    nashorn.eval(getScript("https://fb.me/react-dom-server-15.0.1.js"));
-
-    System.out.println(nashorn.eval("ReactDOMServer.renderToString(React.createElement('div', {}, 'Hello World!'));"));
-  }
+    nashorn.eval(new FileReader("path/to/react.js"));
+    nashorn.eval(new FileReader("path/to/react-dom-server.js"));
```

Comment:
I think we should have a comments indicating where this file can be downloaded, because we don't put the ReactDOMServer build on the downloads page so it's not obvious where to get that file.